crates/dreammaker/src/builtins.rs

```diff
@@ -629,6 +636,22 @@ pub fn register_builtins(tree: &mut ObjectTreeBuilder) {
         list/proc/Swap(Index1, Index2);
         list/var/len;
 
+        // 516
+        alist;
+        alist/var/const/type;
+        alist/var/const/parent_type;
+        alist/var/tag;
```

Comment:
this isn't a datum child, is it?

Reply:
![LXdMDbpbOaeM7y6D@2x](https://github.com/user-attachments/assets/a8533e9e-ddc6-4f9f-b309-3a399ed1bd94)

wouldn't that be nice 🙁

Reply:
hghhhhhhhhhhhhhhhhh

Reply:
the answer is not really, it just exposes a tag var (for some really awful usecases ig) and a const parent/type var, like lists